### PR TITLE
Taiga Issue #72

### PR DESCRIPTION
**Pull request by cecilia-la-place**

### CODE REVIEW

File: src/main/java/net/sf/memoranda/ui/LoginDialog.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        System.exit(0);
+        //TODO - Write what you want to happen if the user hits cancel
+    }
+	
+	void loginB_actionPerformed(ActionEvent e) {
+		if (getSignIn()){
+	        dialog.dispose();
+	    }
+		else{
+			userText.setText("Incorrect login");
```
Comment (PaHorton) [inline src/main/java/net/sf/memoranda/ui/LoginDialog.java]: Can this change something other than the user text. Or perhaps keep the user the same over invalid dialogues. This way I don't have to sit there and backspace "Incorrect login"

File: src/main/java/net/sf/memoranda/ui/LoginDialog.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        //userStr.setForeground(new Color(0, 0, 124));
+        userStr.setHorizontalAlignment(JLabel.CENTER);
+
+        //passStr.setFont(new java.awt.Font("Dialog", 0, 20));
+        //passStr.setForeground(new Color(0, 0, 124));
+        passStr.setHorizontalAlignment(JLabel.CENTER);
+        textPanel.add(userStr);
+        textPanel.add(userText);
+        textPanel.add(passStr);
+        textPanel.add(passText);
```
Comment (PaHorton) [inline src/main/java/net/sf/memoranda/ui/LoginDialog.java]: I would make usertext and passtext select all on focus using 
`phoneNumber.addFocusListener(new java.awt.event.FocusAdapter() {
            public void focusGained(java.awt.event.FocusEvent evt) {
                SwingUtilities.invokeLater(new Runnable() {
                    @Override
                    public void run() {
                        phoneNumber.selectAll();
                    }
                });`

Comment (PaHorton) [inline src/main/java/net/sf/memoranda/ui/LoginDialog.java]: Oh wow... That did not set up properly. I'll format on slack

File: src/main/java/net/sf/memoranda/ui/App.java [hunk 3]
```diff
@@ -135,15 +139,22 @@ void init() {
...
 		frame.dispose();
 	}
-
+	
+	/**
+	 * Initialize the login GUI.
+	 * @return
+	 */
+	private boolean showLogin(){
+		return new LoginDialog().loginSuccess();
```
Comment (James-Quigley) [inline src/main/java/net/sf/memoranda/ui/App.java]: Does this always return false? If login success just returns variable valid, and valid is initialized to false...

If it does, is that the desired outcome?

**Comment by PaHorton** [comment]
To be fixed next sprint. This is just a placeholder.

**Comment by velovix** [comment]
What happened here? It looks like changes from a lot of other commits got pulled in.

**Comment by velovix** [comment]
This PR has been superseded so I'm closing it.

**Pull request closed without merging**